test/functional/lua/treesitter_spec.lua

```diff
@@ -643,6 +641,65 @@ static int nlua_schedule(lua_State *const lstate)
     ]] }
   end)
 
+  it("supports highlighting injected languages", function()
+    local screen = Screen.new(75, 10)
```

Comment:
typically we will share the screen initiation in a `before_each()` block (with a `describe()` to group the highlight related tests)

Reply:
Done!